### DISCUSSION TITLE
Difference in localization performance between two different D455 cameras

### DISCUSSION THREAD

**Discussion by stevemartinov** [stella-cv/stella_vslam#261] [Q&A] [answered]
Hi,

I have two D455 cameras and I am using their factory calibration when running openvslam in localization mode. 

So I have made a map using camera 1 and then tried to localize against the map with the same camera 1 and the results were not that good. It was able to localize but very often the last localization pose right before getting LOST was bad.

Then I used camera 2 and loaded the same map but the results were much better. It was able to localize almost all the time.

I have tried to run 4 times with both cameras to be sure that it is not just a random.

What could be the reason? Can Realsense camera factory calibration might be that off?

**Comment by ymd-stella** [comment, marked as answer]
There must be some different shooting conditions.
